packages/common-lib/src/services/RestClient.ts

```diff
@@ -21,4 +21,9 @@ export async function update(url: string, body: any, headers: any = {}) {
   })
 }
 
-//export async function distory(url:string, body:any) {}
+export async function distory(url: string, body: any, headers: any = {}) {
```

Comment:
"delete" name will be more appropriate and in sync with REST convention as well

Reply:
delete cant be use as a function because delete is Identifier Names